### DISCUSSION TITLE
Icon suggestions

### DISCUSSION THREAD

**Discussion by corsacca** [DiscipleTools/disciple-tools-theme#1208] [Ideas and Requests]
Suggest a new icon and a replacement for an existing icon here.

**Comment by mikeallbutt** [comment]
The icon for a group should represent more than one individual. At present it is the same as a contact record. The same adjustment should done for the name field of a group.

**Reply by corsacca** [reply]
Is this for the icon in the title bar?

**Reply by mikeallbutt** [reply]
Yes that is where I was referring to - in the title bar.

**Reply by mikeallbutt** [reply]
See this issue https://github.com/DiscipleTools/disciple-tools-theme/issues/1261

**Reply by mikeallbutt** [reply]
To keep inline with what the DT app uses, something like this icon would be good:
https://icons.expo.fyi/FontAwesome/group

**Comment by mikeallbutt** [comment]
The `Assigned to` icon (https://icons.expo.fyi/MaterialCommunityIcons/briefcase-account) should be similar to the `Sub-assigned to` icon for a contact record. The `Sub-assigned to` icon could be an outline of the `Assigned to` icon.

I suggest this https://icons.expo.fyi/MaterialCommunityIcons/briefcase-account-outline for the `Sub-assigned to` icon

**Comment by mikeallbutt** [comment]
**Sources** icon could be https://icons.expo.fyi/FontAwesome5/compress-arrows-alt as this better indicates how a source came to the system.

**Comment by mikeallbutt** [comment]
For a contact the **name** could be https://icons.expo.fyi/FontAwesome5/user-alt and **nickname** could be https://icons.expo.fyi/FontAwesome5/user-tag which is similar and has a tag icon (which is like a nickname for someone)

**Comment by mikeallbutt** [comment]
The **gender** icon is a light version and not as bold as the other icons around it. This icon is similar to what is currently being used but is more noticeable: https://icons.expo.fyi/FontAwesome5/transgender

**Comment by mikeallbutt** [comment]
The **Other Social Links** should have an icon that represents links and not the current comment dialogue icon. Maybe this one https://icons.expo.fyi/FontAwesome/link or this bolder/thicker one https://icons.expo.fyi/FontAwesome5/link

**Comment by mikeallbutt** [comment]
The `baptized by` and `baptised who` should have different icons. Options that could be good are:

- https://icons.expo.fyi/MaterialCommunityIcons/water
- https://icons.expo.fyi/Entypo/water
- https://icons.expo.fyi/FontAwesome5/water

**Comment by mikeallbutt** [comment]
The **address** / **locations** fields could use 

- https://icons.expo.fyi/FontAwesome5/directions
- https://icons.expo.fyi/MaterialCommunityIcons/home-map-marker
- https://icons.expo.fyi/FontAwesome/map-marker

**Comment by mikeallbutt** [comment]
**Seeker Path** could use:

- signs like https://icons.expo.fyi/FontAwesome/map-signs
- a path like icon https://icons.expo.fyi/MaterialCommunityIcons/map-marker-path

**Comment by mikeallbutt** [comment]
The **group name** icon in the group details tile would be better with an icon that isn't the same as the icon for a contact record. Something like https://icons.expo.fyi/FontAwesome5/sign would be better.

**Comment by mikeallbutt** [comment]
The **status** of a contact and group do and should use the same icon. Another option would be to use one like a traffic light which suggests go, pause, stop.
https://icons.expo.fyi/FontAwesome5/traffic-light
